UseCase47/src/V1/Worker/Delegate.php

```diff
@@ -0,0 +1,129 @@
+<?php
+declare(strict_types=1);
+
+namespace Neighborhoods\KojoFitnessUseCase47\V1\Worker;
+
+use Neighborhoods\Kojo\Api;
+use Neighborhoods\KojoFitnessUseCase47\V1\Worker;
+
+class Delegate implements DelegateInterface
+{
+    use Worker\Queue\Message\AwareTrait;
+    use Api\V1\Worker\Service\AwareTrait;
+
+    public function businessLogic() : DelegateInterface
+    {
+        $iterations = 0;
+
+        while (true) {
+            // Infinite Loop worker
+            $context = [
+                'event_type' => 'task_status',
+                'top_level_status' => 'success',
+                'use_case' => 'UseCase47',
+                'job_id' => $this->getApiV1WorkerService()->getJobId(),
+                'iterations' => $iterations,
+                'nested_object' => [
+                    'random_letters' => $this->randomLetterString(),
+                    'random_word' => $this->randomWord(),
+                    'random_value' => random_int(1, 100),
+                    'status' => 'success',
+                ],
+            ];
+            $this->getApiV1WorkerService()->getLogger()->notice('I can do this forever', $context);
+
+            $iterations++;
+            sleep(3);
```

Comment:
Oh man, if you were sleeping this long between iterations then there's no way you were hitting the default LA limit of 20